### PR TITLE
Update `readme-score` to avoid vulnerability warning on Github

### PR DESCRIPTION
**Pull request by Palleas**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -20,7 +20,7 @@ gem "sinatra", "1.3.1"
 gem "travis"
 gem "nap"
 gem 'mustache'
-gem 'readme-score', '~> 0.0.2'
+gem 'readme-score', :git => "git@github.com:clayallsopp/readme-score.git", :branch => "master"
```
Comment (DanToml) [inline Gemfile]: could you point this at the SHA to avoid accidentally bumping to an unstable master?

File: Gemfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-source 'https://rubygems.org'
```
Comment (segiddins) [inline Gemfile]: This should be reverted

Comment (Palleas) [reply Gemfile]: Woops, good catch.

**Comment by Palleas** [comment]
@DanToml @segiddins updated based on your feedback.

**Comment by DanToml** [comment]
@Palleas Looks like you need to `bundle update readme-score` again to make CI happy, but otherwise LGTM

**Pull request merged**